packages/pds/tests/views/follows.test.ts

```diff
@@ -34,64 +33,41 @@ describe('pds follow views', () => {
     await close()
   })
 
-  const getCursors = (items: { createdAt?: string }[]) =>
-    items.map((item) => item.createdAt ?? constantDate)
-
-  const getSortedCursors = (items: { createdAt?: string }[]) =>
-    getCursors(items).sort((a, b) => tstamp(b) - tstamp(a))
-
-  const tstamp = (x: string) => new Date(x).getTime()
-
   it('fetches followers', async () => {
     const aliceFollowers = await agent.api.app.bsky.graph.getFollowers(
       { user: sc.dids.alice },
       { headers: sc.getHeaders(alice) },
     )
 
     expect(forSnapshot(aliceFollowers.data)).toMatchSnapshot()
-    expect(getCursors(aliceFollowers.data.followers)).toEqual(
```

Comment:
why'd we ditch this? is it redundant? def appreciate the simplicity tbf

Reply:
Technically it's redundant with the snapshot, but it called out a property of the results a bit more explicitly.  I nixed it because the data to build your own cursor for any of the elements doesn't come down from the API anymore (I think in this case it's based on the `indexedAt` time of the follow record).

Reply:
cool thats what i figured 👍